.tool-versions

```diff
@@ -1,2 +1,3 @@
 golang 1.21.4
 golangci-lint 1.55.2
+mockery 2.38.0
```

Comment:
Is this functional?
We have a make command that installs directly: https://github.com/smartcontractkit/chainlink-common/blob/main/Makefile#L17-L18

Reply:
yeah it works , i use asdf locally so I wanted to install this. it's the same version as the makefile